docs/tutorials/tutorials-runtime-upgrade.md

```diff
@@ -49,6 +48,46 @@ sidebar_label: Runtime Upgrade
 
 * Click the _Submit proposal_ button and sign the transaction
 
+![Submit Proposal](/img/tutorial/submit_proposal.png "Submit Proposal")
+
```

Comment:
a nice to have would be to mention what happens to the proposal if nobody seconds the proposal.
does it automatically get moved to the public referendum anyway?
and how many sessions or eras could go by before that happens, depending on the runtime configuration?